src/components/Header/Header.tsx

```diff
@@ -113,35 +88,9 @@ export function Header(props: HeaderProps) {
 												Paramètres d'affichage
 											</button>
 										</li>
-										{quickAccessItems &&
-											quickAccessItems.map((quickAccessItem, index) => {
-												return (
-													<li key={index}>
-														{quickAccessItem.buttonProps ? (
-															<Button
-																iconId={quickAccessItem.iconId}
-																onClick={quickAccessItem.buttonProps.onClick}
-															>
-																{quickAccessItem.text}
-															</Button>
-														) : (
-															<Button
-																linkProps={{
-																	target: quickAccessItem.linkProps?.target,
-																	href: quickAccessItem.linkProps?.href,
-																	title:
-																		quickAccessItem.linkProps?.target ===
-																		'_blank'
-																			? `${quickAccessItem.text} - ouvre une nouvelle fenêtre`
-																			: quickAccessItem.text,
-																}}
-															>
-																{quickAccessItem.text}
-															</Button>
-														)}
-													</li>
-												);
-											})}
+										{quickAccessItems.map((item, index) => (
+											<QuickAccessLi key={index} item={item} />
```

Comment:
use index as key could cause trouble, use an id or the react hook [`useId`](https://react.dev/reference/react/useId)